### ISSUE TITLE
Copy link process is not intuitive

### ISSUE THREAD

**Issue by ilesinge** [ilesinge/prodo#4]
Copying the link should be more intuitive than right-click, "copy link".
Ideally a click on a button would copy the final URL to the clipboard.

**Comment by raphaelbastide** [comment]
This can be a solution:

```
window.prompt ("Copy to clipboard: Ctrl+C, Enter", "http://copy.me/");
```

…and this is mobile friendly right?

**Comment by ilesinge** [comment]
Should be! I'm trying a js-only copy to clipboard button for now but i'm not sure it's feasible

**Comment by raphaelbastide** [comment]
I also thought about keyboard shortcuts. `CTRL + SHIFT + C` to copy the link URL in the clipboard, something like that. This can solve the click problem. The problem is: it’s a trick, and how to inform the user about this trick?.

**Comment by ilesinge** [comment]
Do you suggest to map a custom function on this key combination? (on my browser it opens "Inspect" mode in developer tools)

**Comment by raphaelbastide** [comment]
Yes, ok it was a bad example. I think it may be a bad idea too. What do you think? `CTRL + ALT + C` maybe?

**Comment by ilesinge** [comment]
The shortcut itself seems OK, but I would prefer to add a button so the feature would be usable on mobile too.

**Comment by raphaelbastide** [comment]
I have a local copy of prodo that uses [clipboard.js](https://zenorocha.github.io/clipboard.js/). And it is pretty handy. Will commit later.

**Comment by raphaelbastide** [comment]
Here is my demo with the `copy` function http://write.raphaelbastide.com/

**Comment by ilesinge** [comment]
Perfect !
